storyblok/storyblok-design-system
# chore: icon updates from Nikola

### Please approve this [PR](https://github.com/storyblok/storyfront/pull/2675) before this, since we will delete an icon on this one

## Pull request type

Jira Link: -



- [x] Other (please describe): Some icons are replaced with new ones

## How to test this PR


Check the app with these icons, you can find the preview link in the storyfront repository as a pull request

![image](https://user-images.githubusercontent.com/11278408/197814814-6eb7ec93-259c-4f33-8024-7bff19c8e11f.png)

You can also check the updated icons in the storybook: https://storyblok-design-system-git-chore-icon-updates-storyblok-com.vercel.app/?path=/docs/design-system-components-sbicon--default#all-icons

## What is the new behavior?



- icons replaces
- 
- 

## Other information
